lib/utils/utilities.js

```diff
@@ -136,6 +136,9 @@ module.exports = {
     PARAMEDIC_DEFAULT_APP_NAME: 'io.cordova.hellocordova',
     SAUCE_USER_ENV_VAR:         'SAUCE_USER',
     SAUCE_KEY_ENV_VAR:          'SAUCE_ACCESS_KEY',
+    SAUCE_HOST:                 'ondemand.saucelabs.com',
+    SAUCE_PORT:                 80,
```

Comment:
These constants also defined in 'wdHelper.js' - consider removing duplicate definition